tedana/decomposition/eigendecomp.py

```diff
@@ -191,62 +196,107 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG, stabilize,
         LGR.info('Loading PCA from: pcastate.pkl')
         with open('pcastate.pkl', 'rb') as handle:
             pcastate = pickle.load(handle)
-        u, s, v = pcastate['u'], pcastate['s'], pcastate['v']
-        ctb, eigelb = pcastate['ctb'], pcastate['eigelb']
-        spmin, spcum = pcastate['spmin'], pcastate['spcum']
+        voxel_comp_weights, varex = pcastate['u'], pcastate['varex']
+        comp_ts = pcastate['comp_ts']
+        ct_df = pcastate['comptable']
+        eigenvalue_elbow = pcastate['eigenvalue_elbow']
+        varex_norm_min = pcastate['varex_norm_min']
+        varex_norm_cum = pcastate['varex_norm_cum']
+
+    np.savetxt('mepca_mix.1D', comp_ts)
+
+    # write component maps to 4D image
+    comp_maps = np.zeros((OCcatd.shape[0], comp_ts.shape[0]))
+    for i_comp in range(comp_ts.shape[0]):
+        temp_comp_ts = comp_ts[i_comp, :][:, None]
+        comp_map = utils.unmask(model.computefeats2(OCcatd, temp_comp_ts, mask), mask)
+        comp_maps[:, i_comp] = np.squeeze(comp_map)
+    utils.filewrite(comp_maps, 'mepca_OC_components.nii', ref_img)
```

Comment:
This is the first time writing out `mepca_OC_components.nii` ? I don't see it elsewhere..

Reply:
It wasn't originally written out, but I thought it might be worth it to do so. Ultimately, when we get around to #17, it would be good to make this output optional (e.g., with `--verbose`).

Reply:
Agreed ! I feel like this could be addressed (at least in part) alongside #106 ...